mapdata/WarcraftLegacies/AbilityData/Skin/808464449.json

```diff
@@ -58,13 +53,13 @@
       "Level": 2,
       "Id": 828536161,
       "Type": 3,
-      "Value": "Instantly teleports 72 of the player\u0027s nearby units, including the caster, to a ground unit or structure in 1250 range."
+      "Value": "After 1.5 seconds, teleports 72 of the player\u0027s nearby units, including the caster, to a ground unit or structure in 1250 range."
```

Comment:
Let's use data fields in these tooltips. I know you can't do it for the delay (because it's a decimal) but it will work for everything else.

Reply:
okay!